sklearn/utils/validation.py

```diff
@@ -232,25 +234,27 @@ def _ensure_sparse_format(spmatrix, accept_sparse, dtype, copy,
     spmatrix_converted : scipy sparse matrix.
         Matrix that is ensured to have an allowed type.
     """
-    if accept_sparse is None:
+    if accept_sparse in [None, False]:
```

Comment:
As I added the possibility to have `accept_sparse=True` so as to accept any sparse format, a user may be tempted to change a `True` into a `False` instead of a `None`